docs/metrics/perplexity.md

```diff
@@ -0,0 +1,118 @@
+---
+search:
+  exclude: true
+---
+
+# Perplexity
+
+!!! info inline end "Requirements"
+
+    It is important to note that perplexity can only be calculated using autoregressive language models.
+
+Perplexity is a metric commonly used in natural language processing to evaluate the quality of language models, particularly in the context of text generation. Unlike metrics such as BLEU or BERT, perplexity doesn't directly measure the quality of generated text by comparing it with reference texts. Instead, perplexity assesses the "confidence" or "surprise" of a language model in predicting the next word in a sequence of words.
```

Comment:
As a general suggestion, docs are easier to review when paragraphs are broken up into multiple lines. I recommend following this repo's standard 120 character line length in docs (currently defined in `.flake8` and applied to code as a pre-commit check).